component/src/main/java/me/androidbox/component/general/TaskButton.kt

```diff
@@ -0,0 +1,64 @@
+package me.androidbox.component.general
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import me.androidbox.component.ui.theme.BusbyTaskyTheme
+import me.androidbox.component.ui.theme.LoginTextColor
+import me.androidbox.component.ui.theme.buttonColor
+
+@Composable
+fun TaskButton(
+    modifier: Modifier = Modifier,
+    buttonText: String,
+    textSize: TextUnit = 16.sp,
+    buttonTextColor: Color = MaterialTheme.colorScheme.LoginTextColor,
+    backgroundColor: Color = MaterialTheme.colorScheme.buttonColor,
+    onButtonClick: () -> Unit,
+    radius: Dp= 38.dp,
```

Comment:
Maybe to be more explicit, I'd call this `borderRadius` since just `radius` sounds like this is a circular button